app/services/migration/migrators/user.rb

```diff
@@ -58,5 +62,20 @@ def validate_multiple_trns!(trns, user)
       user.errors.add(:base, "There are multiple different TRNs from NPQ applications")
       raise ActiveRecord::RecordInvalid, user
     end
+
+    def validate_multiple_emails!(unique_emails, user)
+      return unless unique_emails.size > 1
+
+      user.errors.add(:base, "There are multiple different emails from user identities in NPQ applications")
+      raise ActiveRecord::RecordInvalid, user
+    end
+
+    def validate_existing_email_match!(ecf_email, user)
+      return if user.email.blank?
```

Comment:
I don't think we should need this as NPQ users have to have emails? 🤔

Reply:
This is when a new User is initialized that exists in ECF but doesn't in NPQ with the `ecf_id`, that should not validate.